src/main/java/ua/artcode/billapp/controller/CompanyController.java

```diff
@@ -28,7 +28,7 @@ public CompanyController(CompanyService companyService) {
         try {
             newBill = companyService.createBill(bill);
         } catch (AppException e) {
-            return new ResponseEntity<>("error", HttpStatus.NOT_FOUND);
+            return new ResponseEntity<>("AppException", HttpStatus.INTERNAL_SERVER_ERROR);
```

Comment:
Okay. I get this error. And what next?! No any info about error.
Proper error message (instead "AppException") will be more efficient in this case.
Anyway, forget about exception catching. We will create exception mapper for this.